services/ui-src/src/components/cards/TemplateCard.test.tsx

```diff
@@ -23,26 +23,38 @@ jest.mock("react-router-dom", () => ({
   useNavigate: () => mockUseNavigate,
 }));
 
-const templateVerbiage = verbiage.cards.MCPAR;
+const mcparTemplateVerbiage = verbiage.cards.MCPAR;
+const mlrTemplateVerbiage = verbiage.cards.MLR;
 
-const templateCardComponent = (
+const mcparTemplateCardComponent = (
   <RouterWrappedComponent>
-    <TemplateCard templateName="testTemplate" verbiage={templateVerbiage} />
+    <TemplateCard
+      templateName="testTemplate"
+      verbiage={mcparTemplateVerbiage}
+    />
   </RouterWrappedComponent>
 );
 
+const mlrTemplateCardComponent = (
+  <RouterWrappedComponent>
+    <TemplateCard templateName="MLR" verbiage={mlrTemplateVerbiage} />
+  </RouterWrappedComponent>
+);
+
+mockLDFlags.setDefault({ mlrReport: true });
+
 describe("Test TemplateCard", () => {
   beforeEach(() => {
-    render(templateCardComponent);
+    render(mcparTemplateCardComponent);
   });
 
   test("TemplateCard is visible", () => {
-    expect(screen.getByText(templateVerbiage.title)).toBeVisible();
+    expect(screen.getByText(mcparTemplateVerbiage.title)).toBeVisible();
   });
 
   test("TemplateCard download button is visible and clickable", async () => {
     const apiSpy = jest.spyOn(mockAPI, "getSignedTemplateUrl");
```

Comment:
I read that const as `aSpicy` and was like...what....

Reply:
🤣